community/modules/scheduler/SchedMD-slurm-on-gcp-controller/main.tf

```diff
@@ -58,3 +58,27 @@ module "slurm_cluster_controller" {
   intel_select_solution         = var.intel_select_solution
   cloudsql                      = var.cloudsql
 }
+
+module "slurm_cluster_compute_node" {
+  source  = "github.com/SchedMD/slurm-gcp//tf/modules/compute/?ref=v4.2.0"
+  project = var.project_id
+  cluster_name = (
```

Comment:
nit: extract cluster name logic into locals block and reuse.

Reply:
good point, will update